graylog-project-parent/pom.xml

```diff
@@ -529,6 +529,12 @@
                 <scope>test</scope>
             </dependency>
             <dependency>
+                <groupId>com.github.fakemongo</groupId>
```

Comment:
This is a duplicate: https://github.com/Graylog2/graylog2-server/blob/b9fc16676b6100df5e0c1724f9e61d8831f0dd79/graylog-project-parent/pom.xml#L531-L548